WooCommerce/Classes/Yosemite/DefaultStoresManager.swift

```diff
@@ -351,14 +351,17 @@ private extension DefaultStoresManager {
         }
         dispatch(productSettingsAction)
 
-        group.enter()
-        let sitePlanAction = AccountAction.synchronizeSitePlan(siteID: siteID) { result in
-            if case let .failure(error) = result {
-                errors.append(error)
+        /// skips synchronizing site plan if logged in with WPOrg credentials
+        if siteID != WooConstants.placeholderStoreID {
```

Comment:
is it possible to check this with `isAuthenticatedWithoutWPCom`?

https://github.com/woocommerce/woocommerce-ios/blob/26893e9598f77c721fc553f2cda39e8128c4c02a/WooCommerce/Classes/Yosemite/DefaultStoresManager.swift#L66

Reply:
Good point - also updated in 419896cc7945d582cd75d33f6aeb489d368664fb.